src/com/modbus/modbusclientconnection.cpp

```diff
@@ -14,13 +14,16 @@
 #include "devlog.h"
 #include "modbuspoll.h"
 #include <forte_thread.h>
-#include <unistd.h> // for sleep
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
 #ifndef WIN32
+#include <unistd.h> // for sleep
```

Comment:
I think the better solution for this file is not to fix that here but use `CThread::sleepThread(` instead of sleep. Because the `CThread::sleepThread(` we do the hardware/OS abstraction allready.